BD-3208: Migrating from data feeds to promotion codes (#7611)
braze-inc/braze-docs: _docs/_user_guide/personalization_and_dynamic_content/promotion_codes/migrating_from_data_feeds.md

---
nav_title: Migrating from Data Feeds to Promotion Codes
article_title: Migrating from Data Feeds to Promotion Codes
page_order: 0
description: "This reference article provides guidance on migrating from Data Feeds to promotion codes."
---

# Migrating from Data Feeds to promotion codes

{% alert note %}

> Data Feeds is being deprecated. Braze recommends that customers who use Data Feeds move over to our promotion codes.

Reviewer: ```suggestion
Data Feeds is being deprecated. Braze recommends that customers who use Data Feeds move over to promotion code lists.
```

{% endalert %}

> > Migrating from Data Feeds to promotion codes takes time, but it's a straightforward process! This involves manually creating promotion code lists with the information from your Data Feeds and updating your message references accordingly.

Reviewer: We shouldn't be too excited with this, since users will likely be frustrated with needing to do extra work. This is a much simpler process than migrating from News Feed to Content Cards, so I'd suggest striking the first part.
```suggestion
> Migrating from Data Feeds to promotion codes is a straightforward process. This involves manually creating promotion code lists with the information from your Data Feeds and updating your message references accordingly.
```

> There are two main differences between the promotion codes and Data Feeds:
> 
> - You can use descriptions and expiration dates with promotion codes, not with Data Feeds.
> - You create promotion codes by uploading a CSV, whereas you create Data Feeds by pasting text.

Reviewer: Suggest making this a table instead
```suggestion
## Features and functionality

There are a few differences between promotion code lists and Data Feeds.

| Feature | Promotion codes  | Data Feeds  |
|---|---|---|
| Descriptions          | Yes                                            | No                                     |
| Expiration Dates      | Yes                                            | No                                     |
| Creation Method       | Uploading a CSV                                | Pasting text                           |
{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
{
```

Reviewer: Output that from GPT so you may need to finesse the spacing 🙈

## How to migrate

To replace a Data Feed with a promotion code list, do the following: 

1. Go to **Data Settings** and select **Create Promotion Code List**.

> 2. [Set up your promotion code]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/promotion_codes).

Reviewer: ```suggestion
2. [Set up your promotion code list]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/promotion_codes).
```
